drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -541,6 +560,29 @@ void nrf_802154_cca_failed(nrf_802154_cca_error_t error)
 	k_sem_give(&nrf5_data.cca_wait);
 }
 
+void nrf_802154_energy_detected(uint8_t result)
+{
+	if (energy_scan_done != NULL) {
+		s16_t dbm;
+		energy_scan_done_cb callback = energy_scan_done;
```

Comment:
You need an empty line after declarations.

Reply:
energy_scan_done_cb callback = energy_scan_done; is also a declaration, there is a new line after

Reply:
True, lack of highlighting on GH confused me 😕